src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -169,31 +267,39 @@ export default function TokenTable({ data }: { data: TokenData[] | undefined })
     )
   }
 
-  if (showFavorites && sortedFilteredTokens?.length === 0) {
+  if (showFavorites && topTokens.length === 0) {
     return <NoTokensState message={<Trans>You have no favorited tokens</Trans>} />
   }
 
-  if (!showFavorites && sortedFilteredTokens?.length === 0) {
+  if (!showFavorites && topTokens.length === 0) {
     return <NoTokensState message={<Trans>No tokens found</Trans>} />
   }
 
   return (
-    <Suspense fallback={<LoadingTokenTable />}>
-      <GridContainer>
-        <HeaderRow />
-        <TokenRowsContainer>
-          {sortedFilteredTokens?.map((token, index) => (
-            <LoadedRow
-              key={token.address}
-              tokenAddress={token.address}
-              tokenListIndex={index}
-              tokenListLength={sortedFilteredTokens.length}
-              tokenData={token}
-              timePeriod={timePeriod}
-            />
-          ))}
-        </TokenRowsContainer>
-      </GridContainer>
-    </Suspense>
+    <GridContainer>
+      <HeaderRow />
+      <TokenDataContainer>
+        <AutoSizer>
+          {({ height, width }) => (
+            <InfiniteLoader isItemLoaded={isItemLoaded} itemCount={MAX_TOKENS_TO_LOAD} loadMoreItems={loadMoreItems}>
+              {({ onItemsRendered, ref }: { onItemsRendered: (props: ListOnItemsRenderedProps) => any; ref: any }) => (
+                <FixedSizeList
+                  className="List"
```

Comment:
i think we prefer using `style` vs `className`

Reply:
i removed classname